packages/react-devtools-shared/src/__tests__/console-test.js

```diff
@@ -1042,15 +1043,16 @@ describe('console', () => {
     expect(mockWarn.mock.calls[1][1]).toMatch('warn');
     expect(normalizeCodeLocInfo(mockWarn.mock.calls[1][2]).trim()).toEqual(
       supportsOwnerStacks
-        ? 'in Parent (at **)'
+        ? 'in Object.overrideMethod (at **)' + // TODO: This leading frame is due to our extra wrapper that shouldn't exist.
```

Comment:
This happens because the `patchForStrictMode` methods add an extra stack frame that we don't know we need to pop off.

We should really just get rid of that extra patching because it runs late and is therefore running after potential user space patching.

We should instead reuse the early patching that we already do for the purpose of strict mode printing. That way whatever user spaces wrappers do will be before the dimming.

Reply:
> We should instead reuse the early patching that we already do for the purpose of strict mode printing. That way whatever user spaces wrappers do will be before the dimming and we don't add an extra stack frame here.

Yup. I am planning to work on it this week.